using-mobile-extensions/adobe-campaign-standard/adobe-campaignclassic-api-reference.md

```diff
@@ -0,0 +1,70 @@
+# Campaign API reference
+
+## Set linkage fields
+
+Allows Campaign to connect fields from separate databases and create a more developed and personalized messaging experience. Profile template-based messages that contain PII-based personalization are downloaded. 
```

Comment:
I would refrain from talking about table joins here. If needed, please use correct campaign terminology regarding their app subscriber and recipient tables. You may want to specify the correct template names too here :)